src/blocks/card.css

```
@@ -0,0 +1,91 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+.card {
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  justify-content: center;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+  margin-bottom: 40px;
+}
```

You can center your image without using flex-box container
```suggestion
.card__img {
	display: block;
  height: 134px;
  width: 160px;
  margin: 0 auto 40px;
}
```